lib/p2p/Peer.ts

```diff
@@ -87,9 +87,16 @@ class Peer extends EventEmitter {
     }
   }
 
+  get pubKey(): string {
+    if (this.host) {
+      return this.host.pubKey;
+    } else {
+      return '';
```

Comment:
I think it's better to return `undefined` rather than an empty string, which is a more appropriate indicator that the `pubKey` does not exist.